dependency-check-suppression.xml

```diff
@@ -0,0 +1,82 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<suppressions xmlns="https://jeremylong.github.io/DependencyCheck/dependency-suppression.1.3.xsd">
+    <suppress until="2023-11-09Z">
```

Comment:
Are we going to have to revisit this in 10 days and update these files again?

Reply:
Updated to expire in a month rather than 10 days.